data/Store.js

```diff
@@ -28,9 +28,7 @@ import {Record} from './Record';
 /**
  * A managed and observable set of local, in-memory Records.
  */
-@ReactiveSupport
-@ManagedSupport
-export class Store {
+export class Store extends HoistModel {
```

Comment:
Thoughts on extending `HoistBase` here instead? Any reason to take the extra step of declaring Stores to be Models? 

(No functional difference between the two at this point, right? Unless some other code would be evaluating `isHoistModel` on a store and expecting it to be true in order for some necessary wiring to happen.)